docs/website/docs/walkthroughs/deploy-a-pipeline.md

```diff
@@ -2,4 +2,58 @@
 sidebar_position: 2
 ---
 
-# Deploy a pipeline
\ No newline at end of file
+# Deploy a pipeline
+
+Before you can deploy a pipeline, you will need to 1) [install dlt](../installation.mdx),
+2) [create a pipeline](./create-a-pipeline.md), and 3) sign up for a [GitHub](https://github.com) account, 
+since you will be deploying using [GitHub Actions](https://github.com/features/actions).
+
+## Add your `dlt` project directory to GitHub
+
+If you have not already, you need to initialize a Git repo in your `dlt` project directory and push it to
```

Comment:
I'd at least add an info

If you work in a repository that is already cloned from Github, you are good to go